test/login.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {expect} from 'chai';
+import {shallow} from 'enzyme';
+import {LoginBase} from '../src/Login';
+import theme from '../src/Login/style';
+
+describe('<Login />', () => {
+
+    it('should include username input', () => {
```

Comment:
What kind of validation would we want? I don't know if it's safe to make any assumptions about what usernames should look like for example, except for maybe character length. A lot of that gets into customization which we will need to do, but I think is better served separately.